ajo/personalization-hybrid/public/script.js

```diff
@@ -46,6 +46,13 @@ function sendDisplayEvent(proposition) {
         },
       },
     },
+    data: {
```

Comment:
`in-app-response-format` is only needed for the requests to get personalization.  not needed for display and interact events.